README.md

```diff
@@ -10,8 +10,21 @@ there. You connect HomeAssistant to it and it will do all the parsing.
 ## Wiring
 Wiring is the same as https://github.com/daniel-jong/esp8266_p1meter and also the initial inspiration of using the inverter of the hardware serial.
 
+The hardware pull-up resistor is not necessary on many boards. For example Wemos/Lolin D1 mini (lite) boards with a CH340 uart chip already have a RX pull-up activated by the bootloader.
+
+The ESP8266 can also be powered directly from the P1 port, without external power supply. The following reference wiring diagram shows how:
+![Image](https://github.com/user-attachments/assets/1f973ffe-c296-4723-9260-f466fd31341d)
+
 ## How to use
 1. In the [include-folder](./include) rename the **'secret_template.h'** to **'secrets.h'**, and enter your WiFi credentials.
-2. Install to an ESP8266 board with [PlatformIO](https://platformio.org) using the [pio run -t upload](https://docs.platformio.org/en/latest/core/userguide/cmd_run.html) command
-3. Connect the ESP8266 to your P1 meter as described in `Wiring` above
-4. In HomeAssistant: Add a new [DSMR Slimme Meter integration](https://www.home-assistant.io/integrations/dsmr/) of type Network and enter the IP/Hostname of your ESP8266 and use port 23
+2. If desired, uncomment the STATIC_IP section in **'main.cpp'**.
```

Comment:
I prefer to also have this via an include file, such that the user only needs to edit a simple external file.